src/components/Terminal/Terminal.js

```diff
@@ -41,12 +41,16 @@ const Terminal = ({ address, modelName }) => {
         history.push(`/models/${model}`);
       }
     };
-    const creds = { macaroons };
+    // const creds = { macaroons };
+    // XXX credentials need to be pulled from the store.
+    const credentials = {
+      user: "admin",
+      credentials: "9eef4b03836b2a805dd5a03d43d5f3c0",
```

Comment:
This repo is public - is this a security risk?

Reply:
Well now that you called it out it is!!!

Haha jk :) It's just the randomly generated password from a test controller that no longer exists.